server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -91,7 +91,13 @@ export function create(
 	router.get("/tenants/:id/keys", (request, response) => {
 		const tenantId = request.params.id;
 		const includeDisabledTenant = getIncludeDisabledFlag(request);
-		const tenantP = manager.getTenantKeys(tenantId, includeDisabledTenant);
+		const getPrivateKeys = getPrivateKeysFlag(request);
```

Comment:
nit(readability): might be a personal misunderstanding, but when I read this, I interpreted it as meaning the return value of `getPrivateKeysFlag()` was a function called `getPrivateKeys()`. IMO, changing `getPrivateKeys` to `usePrivateKeys` everywhere, and then changing `getPrivateKeysFlag()` to `getUsePrivateKeysFlagFromRequest()` would improve readability here a lot